recipes/mdal/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+export CXXFLAGS="${CXXFLAGS} -std=c++11"
+if [ "$(uname)" == "Linux" ]; then
+   export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+fi
```

Comment:
This shouldn't be needed as it's part of conda-forges default flags

Reply:
Deleted in next commit